data/json/items/book/misc.json

```diff
@@ -1465,5 +1465,22 @@
     "flags": [ "TRADER_AVOID" ],
     "weight": "120 g",
     "volume": "200 ml"
+  },
+  {
+    "id": "gun_manual",
+    "type": "BOOK",
```

Comment:
book that gives no skill?

Reply:
Yes, these are just manuals about safety/storage/disassembly, shouldn't give any skill.